### PR TITLE
4 packages from OCamlPro/alt-ergo at 2.4.2

### PR DESCRIPTION
**Pull request by Stevendeo**
This pull-request concerns:
-`alt-ergo.2.4.2`: The Alt-Ergo SMT prover
-`alt-ergo-lib.2.4.2`: The Alt-Ergo SMT prover library
-`alt-ergo-parsers.2.4.2`: The Alt-Ergo SMT prover parser library
-`altgr-ergo.2.4.2`: The GUI for the Alt-Ergo SMT prover

---
* Homepage: https://alt-ergo.ocamlpro.com/
* Source repo: git+https://github.com/OCamlPro/alt-ergo.git
* Bug tracker: https://github.com/OCamlPro/alt-ergo/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

**Comment by mseri** [comment]
@kit-ty-kate there is a conflict between cairo2 and lablgtk3. I think we solved a similar issue between lablgtk and lablgtk3 in the past, but I cannot remember what we did right now. Do you remember?

File: packages/alt-ergo-lib/alt-ergo-lib.2.4.2/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# and not alt-ergo-lib.opam which is generated by dune
+tags: "org:OCamlPro"
+
+license: [
+  "OCamlPro Non-Commercial Purpose License, version 1"
+]
+
+build: [
+  ["ocaml" "unix.cma" "configure.ml" name "--prefix" prefix "--libdir" lib "--mandir" man]
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/alt-ergo-lib/alt-ergo-lib.2.4.2/opam:36]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/alt-ergo-parsers/alt-ergo-parsers.2.4.2/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# and not alt-ergo-parsers.opam which is generated by dune
+tags: "org:OCamlPro"
+
+license: [
+  "OCamlPro Non-Commercial Purpose License, version 1"
+]
+
+build: [
+  ["ocaml" "unix.cma" "configure.ml" name "--prefix" prefix "--libdir" lib "--mandir" man]
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/alt-ergo-parsers/alt-ergo-parsers.2.4.2/opam:35]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/alt-ergo/alt-ergo.2.4.2/opam [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+tags: "org:OCamlPro"
+
+license: [
+  "OCamlPro Non-Commercial Purpose License, version 1"
+  "LGPL-2.1-only"
+]
+
+build: [
+  ["ocaml" "unix.cma" "configure.ml" name "--prefix" prefix "--libdir" lib "--mandir" man]
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/alt-ergo/alt-ergo.2.4.2/opam:33]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/altgr-ergo/altgr-ergo.2.4.2/opam [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+# and not altgr-ergo.opam which is generated by dune
+tags: "org:OCamlPro"
+
+license: [
+  "OCamlPro Non-Commercial Purpose License, version 1"
+]
+
+build: [
+  ["ocaml" "unix.cma" "configure.ml" name "--prefix" prefix "--libdir" lib "--mandir" man]
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/altgr-ergo/altgr-ergo.2.4.2/opam:36]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**